action/protocol/staking/vote_bucket.go

```diff
@@ -223,7 +228,7 @@ func calculateVoteWeight(c genesis.VoteWeightCalConsts, v *VoteBucket, selfStake
 	if remainingTime > 0 {
 		weight += math.Log(math.Ceil(remainingTime/86400)*(1+m)) / math.Log(c.DurationLg) / 100
 	}
-	if selfStake && v.AutoStake && v.StakedDuration >= time.Duration(91)*24*time.Hour {
+	if v.isNative() && selfStake && v.AutoStake && v.StakedDuration >= time.Duration(91)*24*time.Hour {
```

Comment:
Here, we just want to ensure that the nft bucket does not calculate selfStake weighting. However, this can also be achieved by ensuring that `selfStake=false` is passed in when bucket is nft. We can revert it.